src/clock/mumeclock.cpp

```diff
@@ -41,57 +41,7 @@ const QMetaEnum MumeClock::s_westronWeekDayNames
 const QMetaEnum MumeClock::s_sindarinWeekDayNames
     = QMetaEnum::fromType<MumeClock::SindarinWeekDayNamesEnum>();
 
-const QHash<QString, MumeTimeEnum> MumeClock::m_stringTimeHash{
```

Comment:
So the big takeaway here is that the game now sends concrete time-of-day events, which eliminates the need for hardcoding zone-specific string "actions"?